code/__HELPERS/text.dm

```diff
@@ -1028,7 +1028,11 @@ GLOBAL_LIST_INIT(binary, list("0","1"))
 
 	return corrupted_text
 
-#define is_alpha(X) ((text2ascii(X) <= 122) && (text2ascii(X) >= 97))
+/// Checks if the char is lowercase
+#define is_lowercase_character(X) ((text2ascii(X) <= 122) && (text2ascii(X) >= 97))
+/// Checks if the char is uppercase
+#define is_uppercase_character(X) ((text2ascii(X) <= 90) && (text2ascii(X) >= 65))
```

Comment:
Can we rely on text2ascii to never change their numerical IDs? If not, we should add a note here about how to look up the ranges.

Reply:
Pretty sure ascii numbers haven't changed since they were made in the 60s